modules/catalog/src/test/java/org/apache/ignite/internal/catalog/CatalogManagerSelfTest.java

```diff
@@ -1388,9 +1432,48 @@ public void testDropZone() {
         assertThat(manager.execute(dropCommand), willThrow(DistributionZoneNotFoundValidationException.class));
     }
 
+    @Test
+    public void testDropDefaultZoneIsRejected() {
```

Comment:
Let's also check if 'old default zone' can be dropped after setting a 'new default zone'.

Reply:
Added such case, in test "testSetDefaultZone"